package.json

```diff
@@ -158,6 +158,7 @@
     "postcss": "^8.4.5",
     "prettier": "~3.6.2",
     "prettier-plugin-prisma": "^5.0.0",
+    "shx": "^0.4.0",
```

Comment:
Why is this dependency needed? I may have overlooked it, but I don't see where it is being used?